components/scientific/gdl/patches/07-objects.cpp.patch

```diff
@@ -0,0 +1,15 @@
+--- gdl-0.9.7/src/objects.cpp.orig	Mon May 29 11:48:57 2017
++++ gdl-0.9.7/src/objects.cpp	Mon May 29 11:50:09 2017
+@@ -935,6 +935,12 @@
+     {
+       return default_num_threads;
+     }
++#elif defined(__sun__)
```

Comment:
Better use function, as /usr/bin/uptime format differs from /usr/gnu/bin/uptime . And anyway you avoid doing several exec syscalls.